python/lsst/obs/base/fitsRawFormatterBase.py

```diff
@@ -197,7 +265,11 @@ def readInfoComponent(self, fileDescriptor, component):
         elif component == "visitInfo":
             return self.makeVisitInfo(self.readMetadata(fileDescriptor))
         elif component == "wcs":
-            return self.makeWcs(self.readMetadata(fileDescriptor))
+            metadata = self.readMetadata(fileDescriptor)
+            obsInfo = ObservationInfo(self.readMetadata(fileDescriptor))
```

Comment:
This is probably beyond the scope of this ticket (unless you're eager to do it), but I'd like to eventually move this call into `FitsExposureFormatter` and pass `obsInfo` (along with `metadata`) to all of these functions.

Reply:
Yes, I think this is broader work than I want to put on this ticket.